internal/installcfg/installcfg_test.go

```diff
@@ -528,20 +572,23 @@ var _ = Describe("ValidateInstallConfigPatch", func() {
 	})
 })
 
-func getInventoryStr(hostname, bootMode string, ipv6only bool) string {
+func getInventoryStr(hostname, bootMode string, ipv4 bool, ipv6 bool) string {
 	inventory := models.Inventory{
 		Hostname: hostname,
 		Boot:     &models.Boot{CurrentBootMode: bootMode},
 		Interfaces: []*models.Interface{
 			{
-				IPV4Addresses: append(make([]string, 0), "10.35.20.10/24"),
-				IPV6Addresses: append(make([]string, 0), "fe80::1/64"),
+				IPV4Addresses: []string{},
+				IPV6Addresses: []string{},
 				MacAddress:    "some MAC address",
 			},
 		},
 	}
-	if ipv6only {
-		inventory.Interfaces[0].IPV4Addresses = nil
+	if ipv4 {
+		inventory.Interfaces[0].IPV4Addresses = append(make([]string, 0), "10.35.20.10/24")
```

Comment:
Is there a reason this isn't just:
```suggestion
		inventory.Interfaces[0].IPV4Addresses = []string{"10.35.20.10/24"}
```

Reply:
Is one preferred over the other? I suspect there are likely compiler things to make them equally efficient...but I feel like I'm missing something.

Reply:
They are indeed equivalent, it's just my syntax is much more convoluted and yours is much more clear :)
Fixed.